src/utils/image.py

```diff
@@ -349,3 +349,12 @@ def clip_area_to_image_bounds(rectangle, image):
         area_start = [min(w, area_start[0]), min(h, area_start[1])]
         area_end = [min(w, area_end[0]), min(h, area_end[1])]
         return [area_start, area_end]
+
+    @staticmethod
+    def rotate(image, rotation, keep_orignal_shape):
+        if keep_orignal_shape:
+            image_shape = image.shape[0:2]
+            image = cv2.rotate(image, rotation)
+            return ImageUtils.resize_to_shape(image,image_shape)
```

Comment:
```suggestion
            return ImageUtils.resize_to_shape(image, image_shape)
```
You should configure your Python file formatter to match black standards `ms-python.black-formatter`